exercises/change/cases_test.go

```diff
@@ -0,0 +1,138 @@
+package change
+
+// Source: exercism/x-common
+// Commit: cda8f98 Create new exercises structure
```

Comment:
Yes, it runs but the  generated cases_test.go test case array is now empty   I had fudged a bit with a mod to the canonical-data.json and I had forgotten about it.  I will make another attempt to handle the new layout of the data and also the difference between a [ list ] or a -1 in the expected field. The list vs. constant just seems wrong because the object type is different, but JSON is crazy flexible in that way I suppose.